experiments/AMIP/components/land/bucket_utils.jl

```diff
@@ -126,8 +126,8 @@ function get_field(bucket_sim::BucketSimulation, ::Val{:energy})
     end
 
     e_per_area .+=
-        -LSMP.LH_f0(bucket_sim.model.parameters.earth_param_set) .*
-        LSMP.ρ_cloud_liq(bucket_sim.model.parameters.earth_param_set) .* bucket_sim.integrator.u.bucket.σS
+        -LP.LH_f0(bucket_sim.model.parameters.earth_param_set) .*
```

Comment:
Not as part of this PR, but we should have a check that `LP.LH_f0` == `CAP.LH_f0` (same for the other params).

Reply:
I opened an issue for this #610